pydata_center/monitoring/tests.py

```diff
@@ -1,3 +1,52 @@
-from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
 
-# Create your tests here.
+from .models import ServerStatus
+
+
+class ReceiveStatusEndpointTests(APITestCase):
+    def setUp(self):
+        self.url = reverse('monitoring:receive_status')
+
+    def test_receive_valid_status(self):
+        valid_data = {
+            'hostname': 'testserver',
+            'ip': '192.168.0.10',
+            'uptime': 34567.8,
+            'timestamp': '2025-05-30T06:27:00Z',
+            'os': 'Ubuntu 22.04',
+            'healthy': True,
+            'server_name': 'DNSServer'
+        }
```

Comment:
Extract `valid_data` to a method or constant. Avoid duplication and ease future changes.
```
def _get_valid_status_data(self):
    return {
        'hostname': 'testserver',
        'ip': '192.168.0.10',
        'uptime': 34567.8,
        'timestamp': '2025-05-30T06:27:00Z',
        'os': 'Ubuntu 22.04',
        'healthy': True,
        'server_name': 'DNSServer'
    }
```